git-extra/astextplain

```diff
@@ -8,6 +8,9 @@ fi
 
 # XXX output encoding (UTF-8) hardcoded
 case "$1" in
+	*.ods | *.ODS | *odf |*.ODF | *odt | *.ODT)
+		odf2txt "$1" || cat "$1"
```

Comment:
I can provide a PKGBUILD for arch linux as I maintain several AUR packages, but need instructions of where to push that file.

An existing example would be great. (Cannot imagine how arch linux relates with git for windows, but really want to known how)